ui/articles-aem-renderer/src/main/java/org/cru/godtools/articles/aem/service/AEMDownloadManger.java

```diff
@@ -147,6 +147,13 @@ private void enqueueExtractAemImportsFromManifests() {
         }
     }
 
+    @AnyThread
+    public void enqueueSyncManifestAemImports(Manifest manifest) {
+        for (Uri uri : manifest.getAemImports()) {
+            enqueueSyncAemImport(uri, true);
```

Comment:
you will want to update `enqueueSyncAemImport` to return a `ListenableFuture` as well. Then use `Futures.successfulAsList` to combine all the Futures into a single Future

Reply:
The force flag should be passed in via the `enqueueSyncManifestAemImports` method.